platform/src/pages/_app.jsx

```diff
@@ -1,5 +1,13 @@
 import '@/styles/global.scss';
+import Head from "next/head";
 
 export default function App({ Component, pageProps }) {
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <link rel="shortcut icon" href="icons/favicon.ico" />
```

Comment:
Thanks, @patrikbodnar . We have a custom Document(_document.jsx) with a Head component where you can add the link. Kindly move that there.

Reply:
Ok, I will do it right now :)

Reply:
@Codebmk , I have moved that there. :)

Reply:
Great @patrikbodnar